bootstrap.py

```diff
@@ -80,9 +80,14 @@
     if boot_branch_file.exists():
         prev_branch = boot_branch_file.read_text()
 
-    result = subprocess.run(
-        ["/usr/bin/env", "git", "merge-base", "--fork-point", "master"]
-    )
+    if os.name == "nt":
+        result = subprocess.run(
+            ["git.cmd", "merge-base", "--fork-point", "master"], shell=True
+        )
+    else:
+        result = subprocess.run(
+            ["/usr/bin/env", "git", "merge-base", "--fork-point", "master"]
+        )
```

Comment:
I get the following error on my Windows VM:
```
'git.cmd' is not recognized as an internal or external command,
operable program or batch file.
```
However, the following does work for me on both macOS and my Windows VM:
```suggestion

    result = subprocess.run(
        ["git", "merge-base", "--fork-point", "master"]
    )
```
I think that the use of `shell=True` is discouraged, as well.

Previously, I used `spyder.utils.programs.find_git` and `spyder.utils.programs.run_program`, which did not use `shell=True`. However, this required that `spyder` be installed, which may not be the case at this point in `bootstrap.py`. 

I'd think the most elegant solution would be to import `spyder.utils.programs` directly using `importlib` tools, except that the module imports from other `spyder` modules as well.

So we may need to duplicate `find_git` here in order to get the correct `git` command on any given system. Furthermore, `find_git` also checked that `git` was properly installed for macOS.

Do you or @ccordoba12 have any better ideas?